web/src/user/settings/sidebaritems.ts

```diff
@@ -29,5 +29,13 @@ export const userSettingsSideBarItems: UserSettingsSidebarItems = {
             to: '/tokens',
             condition: () => window.context.accessTokensAllow !== 'none',
         },
+        {
+            label: 'Permissions',
+            to: '/permissions',
+            exact: true,
+            condition: ({ user, authenticatedUser }) =>
+                !!window.context.site['permissions.backgroundSync']?.enabled &&
+                (authenticatedUser?.siteAdmin || user.id !== authenticatedUser?.id),
```

Comment:
Shouldn't this be equals?
```suggestion
                (authenticatedUser?.siteAdmin || user.id === authenticatedUser?.id),
```

Reply:
Not equal means "it is site admin viewing someone else's settings page". The thing here is that user shouldn't see permissions page themselves.

On a second thought, I think `authenticatedUser?.siteAdmin` alone is sufficient!

Reply:
Oh, I thought it was meant to be "site admins can view everything, regular users only their own data". Seems like siteAdmin should indeed only be checked.

Reply:
>"site admins can view everything, regular users only their own data" 

Yeah, this is true so far for everything else in user settings. Currently only site admin has access to any permissions-related data, here just to be consistent.